lib/ht_sip_validator/validation/sip_validator.rb

```diff
@@ -5,33 +5,23 @@ module Validation
     # Service reponsible for validating a sip
     class SIPValidator
       # Creates a new validator service using the specified configuration
-      #
-      # @param config [String] path to a YAML file with the validator configuration
-      def initialize(config)
-        @config = YAML.load(config)
+      def initialize(validators, logger)
+        @validators = validators
+        @logger = logger
       end
 
       # Validates the given volume and reports any errors
       #
       # @param sip [SubmissionPackage] The volume to validate
       def validate(sip)
-        do_package_checks(sip)
```

Comment:
There was actually a reason for this, although I think the change is fine for now - when we also do image checks, the validate method will also need to run FITS on the relevant files and call the image validators. So the package checks will probably need to be extracted back out at a later date.